ocaml/vhd-tool/src/impl.ml

```diff
@@ -1088,10 +1090,16 @@ let write_stream common s destination destination_protocol prezeroed progress
         x
     | None ->
         let t = List.hd possible_protocols in
-        Printf.fprintf stderr "Using protocol: %s\n%!" (string_of_protocol t) ;
+        D.info "Using protocol: %s\n%!" (string_of_protocol t) ;
         t
   in
-  if not (List.mem destination_protocol possible_protocols) then
+  if
+    not
+      (List.exists
```

Comment:
This is really a hack, it converts any `Nbd export_name` into a dummy_nbd which is just `Nbd ""` (empty string) to check for possible protocols. The intention of this check is to see if nbd is part of the list of possible protocols, no matter what export name it has. Maybe it would be easier to understand if I do a map first?